omero/institution/getting-started.html

```diff
@@ -9,8 +9,109 @@
+                    <li><a href="{{ site.baseurl }}/training/" target="_blank">Get a training session on OMERO or organize one yourself</a></li>
+                </ul>
+            </div>
+        </div>
+        <!-- end What is needed -->
+        <a id="what-is-needed"></a>
+        <hr class="whitespace">
+        <div class="row column text-center">
+            <h3>What is needed to host OMERO?</h3>
+        </div>
+        <hr>
+        <div class="row">
+            <div class="medium">
+                <p>The resources needed for setting up, keeping and maintaining OMERO vary according to your image data sizes and quality, and the infrastructure in your institution. Nevertheless, some general hints are given here as points which you should think about and implement in your concrete case.</p>
+                <p>You can benefit from asking your colleagues at institutions which are already running OMERO. For that, we also list some contacts to regional groups and list of selected institutions which run OMERO in the <a href="#contacts">Key OMERO Contacts</a> section.</p>
+            </div>
+            <div>
+                <h5>Server hardware and environment</h5>
+                <ul class="dev-ops-links">
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/system-requirements.html#hardware" target="_blank">Hardware</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/system-requirements.html#storage" target="_blank">Storage</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/system-requirements.html#ram" target="_blank">RAM</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/system-requirements.html#cpu" target="_blank">CPU</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/system-requirements.html#software" target="_blank">Software dependencies</a></li>
+                </ul>
+                <h5>Server administrator(s)</h5>
+                <ul class="dev-ops-links">
+                    Knowledge on how to run a server on Linux OS is essential. The following items are frequent tasks for server administrators of an OMERO system:
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/index.html" target="_blank">Server installation</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/index.html" target="_blank">Server maintenance</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/index.html" target="_blank">Server upgrades</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/index.html" target="_blank">Server configureation optimization or changes</a></li>
```

Comment:
The first 4 links here are all to the same page.

Reply:
fixed in https://github.com/ome/www.openmicroscopy.org/pull/462/commits/4b1c50e5f13d78797c69cc19369c3ca70684189c